Common/Repository/VirtualDeviceTableStorage.cs

```diff
@@ -43,15 +43,6 @@ public Task<InitialDeviceConfig> GetDeviceAsync(string deviceId)
             return this.GetDeviceAsync(query);
         }
 
-        public Task<InitialDeviceConfig> GetDevice(string deviceId, string hostName)
```

Comment:
this is a simple getdevice function. Why remove it  - may get used in the future ? there are remove and add, getasync methods anyways.